libraries/ui-shared/Form/Checkbox/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@shopgate/pwa-common/components/Grid';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import FormElement from '@shopgate/pwa-ui-shared/FormElement';
+import UICheckbox from '@shopgate/pwa-ui-shared/Checkbox';
+import style from './style';
+
+/**
+ * A component that provides a styled checkbox for user input in material design.
+ */
+class Checkbox extends Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    className: PropTypes.string,
+    errorText: PropTypes.node,
+    label: PropTypes.node,
+    onChange: PropTypes.func,
+  };
+
+  static defaultProps = {
+    className: '',
+    errorText: '',
+    label: '',
+    onChange: () => {},
+  };
+
+  /**
+   * @return {JSX}
+   */
+  render() {
+    const {
+      name, label, onChange, className,
+    } = this.props;
+    return (
+      <FormElement
+        className={className}
+        htmlFor={name}
+        errorText={this.props.errorText}
+        hasUnderline={false}
+        hasPlaceholder={false}
+      >
+        <Grid>
+          <Grid.Item grow={0}>
+            <UICheckbox
```

Comment:
In the design the checkbox is supposed to be orange (primary color). And I also think we should make the label clickable as well, touching the small rectangle could be annoying especially on smaller screens

Reply:
Also the label looks bold in the design